unit/testing-utils/free_form_cmdline.h

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************\
+
+ Module: Unit test utilities
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+#ifndef CPROVER_TESTING_UTILS_FREE_FORM_CMDLINE_H
+#define CPROVER_TESTING_UTILS_FREE_FORM_CMDLINE_H
+
+#include <util/cmdline.h>
+
+/// An implementation of cmdlinet to be used in tests. It does not require
+/// specifying exactly what flags are supported and instead allows setting any
+/// flag
+class free_form_cmdlinet : public cmdlinet
+{
+public:
+  void add_flag(std::string flag);
+  void add_option(std::string flag, std::string value);
+private:
+  void create_flag(const std::string &flag_name);
+};
+
+#endif //CPROVER_TESTING_UTILS_FREE_FORM_CMDLINE_H
```

Comment:
A space after `//` will make the linter happy

Reply:
Fixed in the preceding PR so this will be fixed automatically when #1913 is merged and this is rebased ontop of develop